tests/test_metrics_old.py

```
@@ -0,0 +1,454 @@
+
+        # Verify values are reset (this might not work for all metric types)
+        # At minimum, verify function doesn't crash
+        assert True  # If we get here, function executed
+
+
+class TestRegistryManagement:
+    """Test registry management"""
+
+    def test_registry_is_custom(self):
+        """Test that we use a custom registry"""
+        assert isinstance(REGISTRY, CollectorRegistry)
```

Ruff (0.14.10)

Undefined name `CollectorRegistry`

(F821)